students/AndrewMiotke/lesson02/assignment/src/charges_calc.py

```diff
@@ -0,0 +1,95 @@
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+from logging.config import dictConfig
+
+"""
+NOTES FROM CLASS - DELETE THESE LATER
+Create 2 file handlers
+Create a if/elif statement for each logging level
+"""
+
+# Logging layout
```

Comment:
You will only want 2 handlers, 1 for the console (stream handler) and 1 to write to a file (file handler). The levels for both should be set via the debug argument